pkg/kritis/util/util.go

```diff
@@ -62,12 +63,37 @@ func GetResource(image string) *grafeas.Resource {
 	return &grafeas.Resource{Uri: GetResourceURL(image)}
 }
 
-func GetPgpAttestationFromOccurrence(occ *grafeas.Occurrence) metadata.PGPAttestation {
-	pgp := occ.GetAttestation().GetAttestation().GetPgpSignedAttestation()
-	return metadata.PGPAttestation{
-		Signature: pgp.GetSignature(),
-		KeyID:     pgp.GetPgpKeyId(),
-		OccID:     occ.GetName(),
+func GetRawAttestationFromOccurrence(occ *grafeas.Occurrence) *metadata.RawAttestation {
```

Comment:
nit: we should consider moving this methods to a different package, maybe metadata, i am not sure why it's in util.

Reply:
I can do that if there's no objections. Note we would also have to move `GetVulnerabilityFromOccurrence` as well